openslides_backend/action/actions/mediafile/upload.py

```diff
@@ -51,6 +52,7 @@ class MediafileUploadAction(
     dependencies = [ListOfSpeakersCreate]
 
     def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        add_mimetypes()
```

Comment:
These should be added automatically, an action should not need to know about the installed mimetypes. Maybe you can move this somewhere to the setup of the server?

Reply:
This is done by installing the debian packet `mime-support`. Python mimetypes looks for it, namely the file /etc/mime.types, and uses it for guessing types/extensions. All missed extensions exist with installed mime-support.

Reply:
Perfect, that's the solution I was looking for :D